needy/projects/make.py

```diff
@@ -5,6 +5,18 @@
 from .. import project
 
 
+def get_make_jobs_args(project):
+    jobs = project.needy.jobs()
+    if project.configuration('max-jobs') is not None:
+        jobs = min(jobs, project.configuration('max-jobs'))
```

Comment:
I'm assuming that `make -j 1` is the same as `make`, but I would like to only provide a -j argument in the event that concurrency is actually specified. I agree with `Project::max_concurrency()` though.